src/weapon.py

```diff
@@ -121,7 +123,7 @@ def update(
 
     def shoot(self, projectile_list: arcade.SpriteList):
         rocket = arcade.SpriteSolidColor(30, 20, arcade.color.ORANGE)
-        rocket.position = self.weapon_sprite.position
+        rocket.position = add_vec2(self.weapon_sprite.position, self.muzzle_offset)
```

Comment:
This does not account for rotation of the car.  You can reproduce by point the car up or down and shooting.